google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/stub/GrpcDlpServiceStub.java

```diff
@@ -0,0 +1,241 @@
+      throws IOException {
+    this.operationsStub = GrpcOperationsStub.create(clientContext);
+
+    this.inspectContentCallable =
+        GrpcCallableFactory.create(
+            directInspectContentCallable, settings.inspectContentSettings(), clientContext);
+    this.redactContentCallable =
+        GrpcCallableFactory.create(
+            directRedactContentCallable, settings.redactContentSettings(), clientContext);
+    this.createInspectOperationCallable =
+        GrpcCallableFactory.create(
+            directCreateInspectOperationCallable,
+            settings.createInspectOperationSettings().getInitialCallSettings(),
+            clientContext);
+    this.createInspectOperationOperationCallable =
+        GrpcCallableFactory.create(
+            directCreateInspectOperationCallable,
+            settings.createInspectOperationSettings(),
+            clientContext,
+            this.operationsStub);
+    this.listInspectFindingsCallable =
+        GrpcCallableFactory.create(
+            directListInspectFindingsCallable,
+            settings.listInspectFindingsSettings(),
+            clientContext);
+    this.listInfoTypesCallable =
+        GrpcCallableFactory.create(
+            directListInfoTypesCallable, settings.listInfoTypesSettings(), clientContext);
+    this.listRootCategoriesCallable =
+        GrpcCallableFactory.create(
+            directListRootCategoriesCallable, settings.listRootCategoriesSettings(), clientContext);
+
+    backgroundResources = new BackgroundResourceAggregation(clientContext.getBackgroundResources());
+  }
+
+  public GrpcOperationsStub getOperationsStub() {
+    return operationsStub;
+  }
+
+  public UnaryCallable<InspectContentRequest, InspectContentResponse> inspectContentCallable() {
+    return inspectContentCallable;
+  }
+
+  public UnaryCallable<RedactContentRequest, RedactContentResponse> redactContentCallable() {
+    return redactContentCallable;
+  }
+
+  public OperationCallable<
+          CreateInspectOperationRequest, InspectOperationResult, InspectOperationMetadata,
+          Operation>
+      createInspectOperationOperationCallable() {
+    return createInspectOperationOperationCallable;
+  }
+
+  public UnaryCallable<CreateInspectOperationRequest, Operation> createInspectOperationCallable() {
+    return createInspectOperationCallable;
+  }
+
+  public UnaryCallable<ListInspectFindingsRequest, ListInspectFindingsResponse>
+      listInspectFindingsCallable() {
+    return listInspectFindingsCallable;
+  }
+
+  public UnaryCallable<ListInfoTypesRequest, ListInfoTypesResponse> listInfoTypesCallable() {
+    return listInfoTypesCallable;
+  }
+
+  public UnaryCallable<ListRootCategoriesRequest, ListRootCategoriesResponse>
+      listRootCategoriesCallable() {
+    return listRootCategoriesCallable;
+  }
+
+  /**
+   * Initiates an orderly shutdown in which preexisting calls continue but new calls are immediately
+   * cancelled.
```

Comment:
Actually I'm going to remove this because the overridden method in the interface has docs for this. 

As far as the doc here, it's actually correct - shutdown allows preexisting calls to continue and new calls are cancelled.